### ISSUE TITLE
Janky success and fail messages need an overhaul

### ISSUE THREAD

**Issue by DeanHickerson** [DeanHickerson/Apprise#24]
Validation is appending a message to warn of issues with the form. We should be showing or hiding an existing message. OR we should be removing the message after fade out.

**Comment by DeanHickerson** [comment]
Easy fix:
```
        function dismissPlusTimer($notif) {
            setTimeout(function() {
                if($notif) {
                    $notif.fadeOut(2000, function() {
                        $(this).remove();
                    });
                }
            },3000);
            dismissNotif($notif);
        }

        function dismissNotif($notif) {
            $notif.click(function(){
                $(this).stop().fadeOut(200, function() {
                    $(this).remove();
                });
            });
        }

```

**Comment by DeanHickerson** [comment]
fixed with 2249fc717a90bc6f9bd5fd8b5a27ce9618d5edc4
